src/site/xdoc/server/config-spring-jpa-postgres.xml

```diff
@@ -42,7 +42,7 @@
 
         <p>Example to install it on Debian:</p>
         <source>
-          sudo apt-get update && sudo apt-get install postgresql-9.4 postgresql-client-9.4
+          sudo apt-get update &amp;&amp; sudo apt-get install postgresql-9.4 postgresql-client-9.4
```

Comment:
I don't understand you comment about rebuilding the website. You can keep it in this PR and just put it in a standalone commit, no?